sql/size_utils.sql

```diff
@@ -367,52 +367,48 @@ $BODY$;
 
 -- Convenience function to return approximate row count
 --
--- main_table - hypertable to get approximate row count for; if NULL, get count
---              for all hypertables
+-- main_table - table or hypertable to get approximate row count for
 --
 -- Returns:
--- schema_name      - Schema name of the hypertable
--- table_name       - Table name of the hypertable
--- row_estimate     - Estimated number of rows according to catalog tables
-CREATE OR REPLACE FUNCTION hypertable_approximate_row_count(
-    main_table REGCLASS = NULL
-)
-    RETURNS TABLE (schema_name NAME,
-                   table_name NAME,
-                   row_estimate BIGINT
-                  ) LANGUAGE PLPGSQL VOLATILE
-    AS
+-- Estimated number of rows according to catalog tables
+CREATE OR REPLACE FUNCTION approximate_row_count(main_table REGCLASS)
```

Comment:
the argument is not required to be the main table so should probably be named `relation`